plugins/services/src/js/structs/Framework.js

```diff
@@ -51,4 +45,40 @@ module.exports = class Framework extends Application {
 
     return {value};
   }
+
+  getResources() {
+    // FIXME: Circular reference workaround
+    const MesosStateStore = require('../../../../../src/js/stores/MesosStateStore');
```

Comment:
Previous behavior, counting child tasks as instances, doesn't make sense and is wrong. 
In case MesosState is not there it will calc only the framework itself without its children.

I could get MesosStateStore as an argument but that would break the interface and I'd rather not break it.